packages/neuron-wallet/src/database/chain/subscriber/user-setting-subscriber.ts

```diff
@@ -0,0 +1,57 @@
+import { EntitySubscriberInterface, InsertEvent, RemoveEvent, UpdateEvent } from 'typeorm'
+import { ConnectionName, getConnection, getCurrentConnectionName } from '../connection'
+
+type Constructor<T> = new (...args: unknown[]) => T
+
+export default abstract class UserSettingSubscriber<Entity extends object>
```

Comment:
It will be hard to split client-related and non-client-related data, sometimes we needs to join the table from the DB file. And we will needs to change much codes to adapt the more DB file.